README.md

```diff
@@ -210,6 +210,13 @@ from luxtronik import Luxtronik
 l = Luxtronik('192.168.1.23', 8889, safe=False)
 ```
 
+**CAUTION:** The heat pump controller uses a NAND memory chip which has a
+limited number of erase cycles.
+Since every change of the parameter induces some file changes on the
+controller, frequent parameter changes
+may limit the lifetime of your heat pump, see the discussions in
+[issue 158](https://github.com/Bouni/python-luxtronik/issues/158).
+
```

Comment:
In order to stay consistent with other NOTEs in this README, I would suggest to use `NOTE`. Also writing by itself is not an issue we want to warn about, just the amount of writes, since some changes are needed / wanted.



```suggestion
**NOTE:** The heat pump controller uses a NAND memory chip as persistent storage.
This technology has only a limited number of erase cycles.
As far as we know, every change of a parameter will eventually induce some file changes on the controller,
hence **frequent parameter changes may limit the lifetime of your heat pump**.
For further details refer to [this discussion](https://github.com/Bouni/python-luxtronik/issues/158).
```

Reply:
Thank you for your suggestion. I adopted it, but removed the "as far as we know", since one can be pretty certain that file changes occur (look at the time stamps of `http://your_heat_pump/appl_data1` and `http://your_heat_pump/appl_data2`).